backends/apple/coreml/recipes/coreml_recipe_provider.py

```diff
@@ -111,8 +112,9 @@ def _validate_recipe_kwargs(self, recipe_type: RecipeType, **kwargs: Any) -> Non
 
         unexpected = set(kwargs.keys()) - expected_keys
         if unexpected:
-            raise ValueError(
-                f"Recipe '{recipe_type.value}' received unexpected parameters: {list(unexpected)}"
+            logging.warning(
```

Comment:
Why this turned into warning now? to combine multiple recipes? and fallback to the other backend when it fails?